File: app/Book.php
<?php

namespace App;

use Illuminate\Database\Eloquent\Model;

class Book extends Model {

    protected $fillable = ['id', 'idAvtorja','opisKnjige','cena','naslov'];
    protected $table = 'KNJIGA'; // tukaj določimo dejansko ime tabele po kateri iscemo
}

File: app/src/main/java/ep/rest/Book.kt
package ep.rest

import java.io.Serializable

data class Book(
        val ID: Int = 0,
        val IDAVTORJA: String = "",
        val NASLOV: String = "",
        val OPISKNJIGE: String = "",
        val CENA: Double = 0.0) : Serializable
